### PR TITLE
Update Doc Build Instructions

### PR DESCRIPTION
**Pull request by tst-hrajamoney**

### CODE REVIEW

File: docs/geedocs/docsrc/BUILD_DOCS.md [hunk 1]
```diff
@@ -6,14 +6,12 @@ source files and then generate HTML using Sphinx.
...
-To build the documentation, you need to install Sphinx:
+To build the documentation, the following packages (and versions) need to be installed :
 
-```bash
-sudo apt install python-pip
-sudo pip install sphinx==1.7.5
-sudo pip install GitPython
-sudo pip install sphinxprettysearchresults
-```
+python 2.7 (required)
```
Comment (tst-rwildes) [inline docs/geedocs/docsrc/BUILD_DOCS.md]: required is implied by your previous statement. (Sorry I missed that earlier...)

File: docs/geedocs/docsrc/BUILD_DOCS.md [hunk 2]
```diff
@@ -145,10 +143,12 @@ branch and master.
...
 
-      Merges may become tricky if there are conflicts in the generated HTML.
+      NOTE: Merges may become tricky if there are conflicts in the generated HTML.
       If bringing 5.3.2 changes into master there will be 5.3.2
-      HTML changes. You can keep changes in the 5.3.2 branch while resolving
-      conflicts. Resolve conflicts in the RST and generated HTML for master
+      HTML changes. You can keep 5.3.2 html changes from the 5.3.2 branch while resolving
+      conflicts. If there are conflicts in 5.3.3 html files, regenerating the html files
+      is the best way to solve it.
+      If there are conflicts in the RST, resolve them and generate HTML for master
```
Comment (tst-rwildes) [inline docs/geedocs/docsrc/BUILD_DOCS.md]: take away the 'for master'  These instructions can be followed for any merge into release branches too.  I think we wanted these instructions to be generic for any branch.  Basically only worry about resolving conflicts in non-generated code and then re-generate the generated code to then resolve conflicts in that code.

File: docs/geedocs/docsrc/BUILD_DOCS.md [hunk 3]
```diff
@@ -145,10 +143,12 @@ branch and master.
       Keep the change, which has |release| instead of 5.3.3. This will help
       update the documentation based on the current Open GEE release.
 
-      Merges may become tricky if there are conflicts in the generated HTML.
+      NOTE: Merges may become tricky if there are conflicts in the generated HTML.
       If bringing 5.3.2 changes into master there will be 5.3.2
```
Comment (tst-nfarah) [inline docs/geedocs/docsrc/BUILD_DOCS.md]: consider rewording to make it version generic. May want to edit as "If bringing for example 5.3.2"... or some thing similar

File: docs/geedocs/docsrc/BUILD_DOCS.md [hunk 4]
```diff
@@ -145,10 +143,12 @@ branch and master.
...
-      HTML changes. You can keep changes in the 5.3.2 branch while resolving
-      conflicts. Resolve conflicts in the RST and generated HTML for master
+      HTML changes. You can keep 5.3.2 html changes from the 5.3.2 branch while resolving
+      conflicts. If there are conflicts in 5.3.3 html files, regenerating the html files
```
Comment (tst-nfarah) [inline docs/geedocs/docsrc/BUILD_DOCS.md]: is 5.3.3 master in this example ? If yes i would use 'master' just to make it consistent with previous text that mention master.

File: docs/geedocs/docsrc/BUILD_DOCS.md [hunk 5]
```diff
@@ -145,10 +143,12 @@ branch and master.
...
+      is the best way to solve it.
+      If there are conflicts in the RST, resolve them and generate HTML
```
Comment (tst-nfarah) [inline docs/geedocs/docsrc/BUILD_DOCS.md]: I would define RST if not defined yet.

File: docs/geedocs/docsrc/BUILD_DOCS.md [hunk 6]
```diff
@@ -6,14 +6,12 @@ source files and then generate HTML using Sphinx.
```
Comment (tst-nfarah) [inline docs/geedocs/docsrc/BUILD_DOCS.md]: `To update this documentation`  --> `To update Open GEE documentation`

File: docs/geedocs/docsrc/BUILD_DOCS.md [hunk 7]
```diff
@@ -6,14 +6,12 @@ source files and then generate HTML using Sphinx.
...
-```bash
-sudo apt install python-pip
-sudo pip install sphinx==1.7.5
-sudo pip install GitPython
-sudo pip install sphinxprettysearchresults
-```
+python 2.7
+git 1.8+
+python-pip
+python packages: sphinx(1.7.5), GitPython(1.0.1), sphinxprettysearchresults(0.3.5)
```
Comment (tst-nfarah) [inline docs/geedocs/docsrc/BUILD_DOCS.md]: On [the rendered md](https://github.com/google/earthenterprise/blob/91ad0c5a6bf383928f4a8dd4caed1a62f7ebcb40/docs/geedocs/docsrc/BUILD_DOCS.md) the listed packages appear on same line. If u want them to be listed in separate line, may want to add an empty line i believe.

File: docs/geedocs/docsrc/BUILD_DOCS.md [hunk 8]
```diff
@@ -73,26 +71,22 @@ _fusionTutorial_ folder.
 
     Commit your changes and create a pull request.
```
Comment (tst-nfarah) [inline docs/geedocs/docsrc/BUILD_DOCS.md]: Consider making previous lines (61 ... 68) generic for the version as well.

File: docs/geedocs/docsrc/BUILD_DOCS.md [hunk 9]
```diff
@@ -73,26 +71,22 @@ _fusionTutorial_ folder.
...
 
-## Merging Your Changes with Master
+## Making doc Changes in a release folder and merging into Master
 
-Assume you have branched to release and your release branch is release_5.3.2
-and you are making changes that need to go in to the release_5.3.2
-branch and master.
-
-1. Create a branch off release_5.3.2 branch
+1. Create a branch off release_5.x.y branch
```
Comment (tst-nfarah) [inline docs/geedocs/docsrc/BUILD_DOCS.md]: should we add at the end `in your Fork` ?

**Pull request merged**